app/main/index.ts

```diff
@@ -182,6 +182,10 @@ app.on('ready', () => {
 		} else {
 			mainWindow.show();
 		}
+		if (!ConfigUtil.isConfigItemExists('userAgent')) {
```

Comment:
@akashnimare 
The flaw is here I think. On upgradation, UA would _exist_ in settingsJSON and thus will not be updated